pkg/standalone/testdata/runfileconfig/test_run_config_empty_app_dir.yaml

```diff
@@ -11,8 +11,8 @@ common:
       value: sts
 apps:
   - app_dir_path: ./webapp/
```

Comment:
Suggestion: This should be called something like `app_root`, for users to understand intuitively that `resources_path` and `config_file` are relative to the root. As of now,
1. `app_dir_path` is relative to dapr.yaml
2. `resources_path` and `config_file` are relative to `app_dir_path`

Reply:
For this one can we discuss in #1143 ?

Reply:
Sure, thanks @mukundansundar!